install_files/ansible-base/roles/postfix/templates/main.cf

```diff
@@ -62,3 +62,7 @@ maximal_queue_lifetime = 14d
 # Used to remap outbound from address in emails
 smtp_generic_maps = hash:/etc/postfix/generic
 {% endif %}
+{% if securedrop_target_distribution == "focal" %}
```

Comment:
In it's current form, this condition will never be met in production context on a Focal install. This is because `securedrop_target_distribution` defaults to `xenial` in https://github.com/freedomofpress/securedrop/blob/2de09b4c5b1bf42deaa4331ac33915b5752d98ed/install_files/ansible-base/group_vars/all/securedrop#L29 In staging context, is it overriden in https://github.com/freedomofpress/securedrop/blob/2de09b4c5b1bf42deaa4331ac33915b5752d98ed/molecule/libvirt-staging-focal/ansible-override-vars.yml#L10 This means that while CI/tests are passing (they are using staging environments), the logic as written here will not work in production VMs nor hardware; the condition tested will never be met since `securedrop_target_distribution` is undefined. Using `ansible_distribution_release` should work here.